cmd/loom/chainconfig/cmd.go

```diff
@@ -324,6 +328,136 @@ func RemoveFeatureCmd() *cobra.Command {
 	return cmd
 }
 
+const setValidatorInfoCmdExample = `
+loom chain-cfg set-validator-info --build 1000 --key path/to/private_key
+`
+
+func SetValidatorInfoCmd() *cobra.Command {
+	var flags cli.ContractCallFlags
+	buildNumber := uint64(0)
+	cmd := &cobra.Command{
+		Use:     "set-validator-info",
+		Short:   "Set validator informations",
+		Example: setValidatorInfoCmdExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			request := &cctype.SetValidatorInfoRequest{
+				BuildNumber: buildNumber,
+			}
+			err := cli.CallContractWithFlags(&flags, chainConfigContractName, "SetValidatorInfo", request, nil)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	cli.AddContractCallFlags(cmd.Flags(), &flags)
+	cmdFlags := cmd.Flags()
+	cmdFlags.Uint64Var(&buildNumber, "build", 0, "Set a validator's information ")
+	return cmd
+}
+
+const getValidatorInfoCmdExample = `
+loom chain-cfg get-validator-info 0x7262d4c97c7B93937E4810D289b7320e9dA82857
+`
+
+func GetValidatorInfoCmd() *cobra.Command {
+	var flags cli.ContractCallFlags
+	cmd := &cobra.Command{
+		Use:     "get-validator-info",
+		Short:   "Get validator informations",
+		Example: getValidatorInfoCmdExample,
+		Args:    cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var resp cctype.GetValidatorInfoResponse
+			addr, _ := cli.ParseAddress(args[0], flags.ChainID)
+
+			if err := cli.StaticCallContractWithFlags(&flags,
+				chainConfigContractName,
+				"GetValidatorInfo",
+				&cctype.GetValidatorInfoRequest{Address: addr.MarshalPB()},
+				&resp,
+			); err != nil {
+				return err
+			}
+
+			out, err := formatJSON(&resp)
+			if err != nil {
+				return err
+			}
+			fmt.Println(out)
+			return nil
+		},
+	}
+	cli.AddContractStaticCallFlags(cmd.Flags(), &flags)
+	return cmd
+}
+
+const listValidatorsInfoCmdExample = `
+loom chain-cfg list-validators-info 
+`
+
+func ListValidatorsInfoCmd() *cobra.Command {
+	var flags cli.ContractCallFlags
+	cmd := &cobra.Command{
+		Use:     "list-validators-info",
+		Short:   "list validator informations",
+		Example: listValidatorsInfoCmdExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var resp cctype.ListValidatorsInfoResponse
+			err := cli.StaticCallContractWithFlags(&flags, chainConfigContractName, "ListValidatorsInfo",
+				&cctype.ListValidatorsInfoRequest{}, &resp)
+			if err != nil {
+				return err
+			}
+			out, err := formatJSON(&resp)
+			if err != nil {
+				return err
+			}
+			fmt.Println(out)
+			return nil
+		},
+	}
+	cli.AddContractStaticCallFlags(cmd.Flags(), &flags)
+	return cmd
+}
+
+const sumValidatorsInfoCmdExample = `
+loom chain-cfg sum-validators-info 
+`
+
+func SumValidatorsInfoCmd() *cobra.Command {
```

Comment:
Don't need a separate command for the summary, just print the summary at the end of the validator list printed out by the `list-validators-info` command